src/sentry/conf/server.py

```diff
@@ -808,6 +808,8 @@ def create_partitioned_queues(name):
     "organizations:advanced-search": True,
     # Enable obtaining and using API keys.
     "organizations:api-keys": False,
+    # Enable customization of issue alerts.
+    "organizations:customizable-issue-alerts": False,
```

Comment:
`custom-issue-alerts` says the same thing with fewer letters.

Reply:
Thanks! Sounds good (-:

Unfortunately, I misunderstood the specs the first time round.  Also,  I'm being switched to a different project for the time being. I might get back to this / close this PR.

Reply:
Adopted custom, but added new-project as what the feature affects is ambiguous without it.